routes/celebrity-router.js

```diff
@@ -0,0 +1,52 @@
+const express = require("express");
+
+const Celebrity = require("../models/celebrity.js");
+
+// create the router object (container of some routes)
+const router = express.Router();
+
+// ROUTES go here
+
+router.get("/celebrities", (req, res, next) => {
+  // whenever a user visits "/books" find all the books sorted by rating
+  Celebrity.find()
+    .then(celebrityResults => {
+      // send the database query results to the HBS file as "celebrityArray"
+      res.locals.celebrityArray = celebrityResults;
+      res.render("celebrities/index.hbs");
+    })
+    // next(err) skips to the error handler in "bin/www" (error.hbs)
+    .catch(err => next(err));
+});
+
+// #4 Adding new celebrities
+// move this above the celebrityID router, otherwise it would call the celebrity ID page every time
+router.get("/celebrities/new", (req, res, next) => {
+  res.render("celebrities/new.hbs");
+});
+router.post("/process-add", (req, res, next) => {
+  const { name, occupation, catchPhrase } = req.body;
+
+  Celebrity.create({ name, occupation, catchPhrase })
+    .then(celebrityDoc => {
+      res.redirect("celebrities/show.hbs");
```

Comment:
The reason your redirect doesn't work is because you want to redirect to a route. If you wanted to route back to the celebrities page it would be `res.redirect("/celebrities"); `

we only include the `.hbs` when you want to render a page